src/structures/Client.ts

```diff
@@ -0,0 +1,13 @@
+import { createLogger } from "../utils/Logger";
+import * as config from "../config";
+
+export class Client {
+    public readonly config = config;
+    public readonly logger = createLogger();
+    // eslint-disable-next-line
+    constructor() {}
```

Comment:
```suggestion
    // eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/no-useless-constructor
    public constructor() {}
```
Only disable specific rules